docs/bridge_service.md

```diff
@@ -97,7 +97,66 @@ sequenceDiagram
 This paragraph explains a set of endpoints, that are necessary in order to conduct the bridge flow described above.
 
 ### Get bridges
-<!-- TODO: @temaniarpit27 -->
+Retrieves the bridge(s) for a specified network, supporting pagination. The bridges represent the `BridgeEvent` events emitted by the bridge contract.
+
+#### Parameters
+
+| **Name**           | **Type**    | **Description**                                             | **Required** | **Notes**                       |
+|--------------------|-------------|-------------------------------------------------------------|--------------|---------------------------------|
+| `networkID`        | `uint32`    | ID of the network to fetch token mappings from.             | Yes          | `0` for L1(otherwise L2)        |
+| `pageNumber`       | `*uint32`   | Page number for pagination (pointer to `uint32`).           | No           | Defaults if `nil`.              |
+| `pageSize`         | `*uint32`   | Number of items per page (pointer to `uint32`).             | No           | Defaults if `nil`.              |
+| `depositCount`     | `*uint32`   | Query param for a specific Deposit Count (pointer to `uint32`).             | No           | Defaults if `nil`.              |
```

Comment:
Please format the table (align the columns), based on the last row, which is the largest one.

Reply:
I checked the md. Things seemed to be aligned
![Uploading Screenshot 2025-02-24 at 7.19.59 PM.png…]()

Reply:
I was referring to the markdown code, as it is easier to follow when everything is aligned and each row is a one-liner:

![image](https://github.com/user-attachments/assets/ba40d530-1d87-457b-b83a-878707911011)